src/test/client.js

```diff
@@ -344,6 +330,7 @@ function simulateConnection(dumpPath, resultPath, ua, protocolV) {
 function runTest() {
     testCheckRouter = new TestCheckRouter(server);
 
+    /*
```

Comment:
Maybe add a small comment explaining why this code is commented out?

```suggestion
    /* the result format has changed and we are no longer maintaining the legacy stats.
```

Reply:
Good point. I added a comment.